messages/run.js

```diff
@@ -28,20 +23,26 @@ module.exports = {
 		"vceDescription": "throws an error when violations are detected",
 		"vceDescriptionLong": "Throws an error when violations are detected. Exit code is the most severe violation.",
 		'engineDescription': "engine(s) to run",
-		'engineDescriptionLong': "One or more engines to run. Multiple values can be specified as a comma-separated list."
+		'engineDescriptionLong': "One or more engines to run. Multiple values can be specified as a comma-separated list.",
+		'eslintConfigDescription': 'location of eslintrc config to customize eslint engine',
+		'eslintConfigDescriptionLong': 'Location of eslintrc to customize eslint engine',
+		'pmdConfigDescription': 'location of PMD rule reference XML file to customize rule selection',
+		'pmdConfigDescriptionLong': 'Location of PMD rule reference XML file to customize rule selection'
 	},
 	"validations": {
 		"mustTargetSomething": "Please specify a codebase using --target.", // TODO: Once --org is implemented, rewrite this message.
 		"outfileFormatMismatch": "Your chosen format %s does not appear to match your output file type of %s.",
 		"outfileMustBeValid": "--outfile must be a well-formed filepath.",
-		"outfileMustBeSupportedType": "--outfile must be of a supported type. Current options are .xml and .csv."
+		"outfileMustBeSupportedType": "--outfile must be of a supported type. Current options are .xml and .csv.",
+		"tsConfigEslintConfigExclusive": "Please provide tsconfig path within the eslint config file under 'parseOptions.project'."
 	},
 	"output": {
 		"noViolationsDetected": "No rule violations found.",
 		"invalidEnvJson": "--env parameter must be a well-formed JSON.",
 		"writtenToOutFile": "Rule violations have been written to %s.",
 		"sevDetectionSummary": "Detected rule violations of severity %s or lower.",
-		"pleaseSeeAbove": "Please see the logs above."
+		"pleaseSeeAbove": "Please see the logs above.",
+		"filtersIgnoredCustom": "Rule filters will be ignored for custom config runs. Please modify your config file to reflect the filtering you need."
```

Comment:
Will the user know what a `custom config run` is? Could this be `Rule filters are ignored when --eslintconfig or --pmdconfig are provided`.

Reply:
Good call. Fixing this.

Reply:
But we'll ignore the filter only for the engine in question. For example, a user could've provided `--eslintconfig` and some category filters. We'll still use the category filters for running PmdEngine even if we don't use it for CustomEslintEngine.